src/popover/index.js

```diff
@@ -46,6 +48,50 @@ class Popover extends Component {
   }
 
   render() {
+    return (
+      <span style={{position: 'relative', display: 'inline-block'}}>
+        {this._getElement()}
+        {this._getPopoverElement()}
+      </span>
+    );
+  }
+
+  _getElement = () => {
+    return React.cloneElement(this.props.element, { ref: 'element' });
+  }
+
+  _getPopoverElement = () => {
+    if(!this.props.isOpen)
+      return null;
+    return (
+      <span>
+        {this._getBackDropElement()}
+        {this._getArrowElement()}
+        {this._getTetherElement()}
+      </span>
+    );
+  }
+
+  _getArrowElement = () => {
+    return (
+      <PopoverArrow
+        position={this.props.position}
+        shouldHaveBorder={this._shouldHaveBorder()}
+        style={this.props.style}/>
+    );
+  }
+
+  _getBackDropElement = () => {
+    if( !this.props.onClickOut )
+      return null;
+    return (
+      <div className='pt-popover-backdrop'
+        onClick={this._handleBackDropClick}>
```

Comment:
Hmm, no i suppose it wouldn't. Since the element takes up the full screen, event bubbling doesn't really work. All the user could do would be specify to close the popover...